src/pilot/customize_image.sh

```diff
@@ -53,13 +53,12 @@ fi
 
 repos=(
     rhel-7-server-rpms
-    rhel-7-server-rhscon-2-agent-rpms
     rhel-7-server-rhceph-2-mon-rpms
 )
 
 packages=(
     calamari-server
```

Comment:
We still need calamari server for cephmetrics?

Reply:
That has yet to be determined.  I left it in during my investigation.  I will verify if it's needed during the integration effort.